bin/compile

```diff
@@ -35,16 +38,38 @@ cp -n $BP_DIR/scripts/Procfile $BUILD_DIR/Procfile
 echo "-----> Copying static.json to application slug"
 cp -n $BP_DIR/scripts/static.json $BUILD_DIR/static.json
 
-echo "-----> Installing Bower package manager"
 cd $BUILD_DIR
-npm install bower
+if [ -d "./bower_components" ]; then
+  echo "-----> Relying on cached bower_components"
+else
+  if [ -x $LOCAL_BOWER_PATH ]; then
```

Comment:
This var is hardcoded above. The `elif` and `else` clauses here would never run?